test/inductor/test_torchinductor_codegen_dynamic_shapes.py

```diff
@@ -195,6 +195,9 @@ def run(*ex, **kwargs):
     "test_view_detach_dynamic_shapes": TestFailure(("cpu", "cuda")),
     "test_view_on_aliased_dynamic_shapes": TestFailure(("cpu", "cuda")),
     "test_linear_float64_dynamic_shapes": TestFailure(("cpu")),
+    "test_adaptive_avg_pool_with_output_size_0_dynamic_shapes": TestFailure(
+        ("cpu", "cuda")
+    ),
```

Comment:
what's the problem with dynamic shape?

Reply:
Since we directly return empty when the output size of adaptive_avg_pool is zero, no for loop is generated.
https://github.com/pytorch/pytorch/blob/0ccdbbe2337b0955a41407a7b31991772387abb0/test/inductor/test_torchinductor_codegen_dynamic_shapes.py#L142